src/Bicep.LangServer/ParamsHandlers/BicepParamsTextDocumentSyncHandler.cs

```diff
@@ -1,81 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-using Bicep.Core;
-using Bicep.Core.Features;
-using Bicep.LanguageServer.CompilationManager;
-using Bicep.LanguageServer.Utils;
-using MediatR;
-using OmniSharp.Extensions.LanguageServer.Protocol;
-using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
-using OmniSharp.Extensions.LanguageServer.Protocol.Document;
-using OmniSharp.Extensions.LanguageServer.Protocol.Models;
-using OmniSharp.Extensions.LanguageServer.Protocol.Server.Capabilities;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-
-namespace Bicep.LanguageServer.ParamsHandlers
-{
-    internal class BicepParamsTextDocumentSyncHandler : TextDocumentSyncHandlerBase
```

Comment:
>BicepParamsTextDocumentSyncHandler [](http://example.com/codeflow?start=19&length=34)

The LSP doesn't carry language ID on document change events, so it's better to have one handler to handle text changes.